src/hotspot/share/runtime/abstract_vm_version.hpp

```diff
@@ -44,6 +44,29 @@ typedef enum {
 class outputStream;
 enum class vmIntrinsicID;
 
+#define MAX_FEATURE_VEC_SIZE 4
+
+class VM_Features {
+ public:
+  using FeatureVector = uint64_t [MAX_FEATURE_VEC_SIZE];
```

Comment:
Why did you decide to declare new type name for fixed size array type? I see you use `FeatureVector` in `vmStructs*` and JVMCI code. Does it make things simpler there?

Reply:
Yes. I was facing compilation issues with raw array types.